packages/tldraw/src/lib/ui/components/StylePanel/DefaultStylePanelContent.tsx

```diff
@@ -123,64 +123,72 @@ export function CommonStylePickerSet({ styles, theme }: ThemeStylePickerSetProps
 
 	return (
 		<>
-			<div tabIndex={-1} className="tlui-style-panel__section__common" data-testid="style.panel">
+			<div className="tlui-style-panel__section__common" data-testid="style.panel">
 				{color === undefined ? null : (
-					<TldrawUiButtonPicker
-						title={msg('style-panel.color')}
-						uiType="color"
-						style={DefaultColorStyle}
-						items={STYLES.color}
-						value={color}
-						onValueChange={handleValueChange}
-						theme={theme}
-						onHistoryMark={onHistoryMark}
-					/>
+					<TldrawUiToolbar label={msg('style-panel.color')}>
```

Comment:
i thought we had discussed making these radio groups instead of toolbars so it's easier to change the values within them and navigate between the different controls?

Reply:
It's also a bit odd to me that these are now all announced as toolbars.

For colors, it's surprising that the up/down arrow keys move my selection right/left

Reply:
yeah, couple things here that made me rethink the radio vs. toolbar thing:
- the main reason was that the Radix UI component for radio/radiogroups didn't fit right: https://www.radix-ui.com/primitives/docs/components/radio-group they have items/indicators and it felt awkward with what we were using inputs/indicators/labels (well, we'd leave out the labels). yeah i'm sure we could restyle it all to look like buttons but it just felt awkward.
- furthermore, looking a bit at Miro made me think it was more natural to use buttons than to use the radio/input paradigm.

but let's talk further if you think we should think about this some more.

Reply:
I messed around with this some more today to see how it felt like, and ach, maybe?
To be honest, it's very easy to swap out for the RadioGroup if we want. It feels like it might be one of those things that our a11y auditor/consultant will say: we should use RadioGroup vs. Toolbar. I would trust their opinion on which way to go more than ours maybe :)

Reply:
I'd love to have a play with the radio button version if u have a link